hawc/apps/riskofbias/serializers.py

```diff
@@ -39,11 +39,10 @@ class Meta:
 
 class SimpleRiskOfBiasMetricSerializer(serializers.ModelSerializer):
     response_values = serializers.ListField(source="get_response_values")
-    default_response = serializers.IntegerField(source="get_default_response")
```

Comment:
@rabstejnek I removed this since it's no longe needed on the form side; let me know if you still need it and I'm missing something